other/verify_image.yaml

```diff
@@ -16,18 +16,19 @@ metadata:
       its signature against the provided public key. This policy serves as an illustration for
       how to configure a similar rule and will require replacing with your image(s) and keys.
 spec:
-  validationFailureAction: enforce
+  validationFailureAction: audit
```

Comment:
I realize we want to default to audit, but it seems like enforce would be a better default for image verification policies, as most users will expect unsigned images to be blocked.

One way to balance, is to use specific image registries instead of "*".

Reply:
Set all 3 policies to `enforce`. See latest commit.